loadtest/user/userentity/utils.go

```diff
@@ -22,3 +27,107 @@ func postListToSlice(list *model.PostList) []*model.Post {
 	}
 	return posts
 }
+
+func closeBody(r *http.Response) {
+	if r.Body != nil {
+		_, _ = io.Copy(io.Discard, r.Body)
+		_ = r.Body.Close()
+	}
+}
+
+func convertToTypedPrefs(input []gqlPreference) model.Preferences {
+	prefs := model.Preferences{}
+	for _, p := range input {
+		prefs = append(prefs, model.Preference{
+			UserId:   p.UserID,
+			Category: p.Category,
+			Name:     p.Name,
+			Value:    p.Value,
+		})
+	}
+	return prefs
+}
+
+func convertToTypesRoles(input []gqlRole) []*model.Role {
+	roles := []*model.Role{}
+	for _, r := range input {
+		roles = append(roles, &model.Role{
+			Id:            r.ID,
+			Name:          r.Name,
+			Permissions:   r.Permissions,
+			SchemeManaged: r.SchemeManaged,
+			BuiltIn:       r.BuiltIn,
+			CreateAt:      int64(r.CreateAt),
+			DeleteAt:      int64(r.DeleteAt),
+			UpdateAt:      int64(r.UpdateAt),
+		})
+	}
+	return roles
+}
+
+func convertToTypedUser(input gqlUser) (*model.User, model.Preferences, []*model.Role) {
+	prefs := convertToTypedPrefs(input.Preferences)
+	roles := convertToTypesRoles(input.Roles)
+	user := &model.User{
+		Id:            input.ID,
+		Username:      input.Username,
+		Email:         input.Email,
+		FirstName:     input.FirstName,
+		LastName:      input.LastName,
+		CreateAt:      int64(input.CreateAt),
+		UpdateAt:      int64(input.UpdateAt),
+		DeleteAt:      int64(input.DeleteAt),
+		EmailVerified: input.EmailVerified,
+		IsBot:         input.IsBot,
+		Timezone:      input.Timezone,
+		Props:         input.Props,
+		NotifyProps:   input.NotifyProps,
+	}
+
+	return user, prefs, roles
+}
+
+func convertToTypedTeams(userID string, input []gqlTeamMember) ([]*model.Team, []*model.TeamMember) {
+	teams := []*model.Team{}
+	tms := []*model.TeamMember{}
```

Comment:
`tms` is a convention we use in the server as well. Since the variable is just used in the same function, it goes with the convention of using [short variable names](https://github.com/golang/go/wiki/codereviewcomments#variable-names).